docs/adr/decisions/0012-add-flow-collections.md

```diff
@@ -0,0 +1,90 @@
+---
+status: "proposed"
+---
+# Add collections to flow and source metadata schemas
+
+## Context and Problem Statement
+
+For optimum flexibility, media Flows (and their parent Sources) are defined as mono-essence.
+However, for all practical purposes, a means of describing how separate mono-essence Flows relate together is needed.  
+As an example, media is often streamed as a multiplex of audio, video and ancillary essence such as subtitles.  
+These elements are intended to be presented together in sync.  
+When ingested into a TAM store, these media elements may be stored separately, with their synchronisation relationship communicated through timestamps placing them on a common timeline.  
+Each element has a separate Flow ID and descriptor.  
+A client of the API needs to be able to discover the fact that these elements were ingested as a group.  
+Beyond this, it would be useful to be able to define new collections or presentation groups built from elements that share a common timeline.  
+For example, commentary or subtitles may be added as new or replacement elements after the initial ingest of an AV feed.
+
+## Considered Options
+
+* Option 1: Delegate the handling of grouping relationships to a separate service with its own API
+* Option 2: Add multi-essence Sources that register collections of Sources in the TAMS API
+* Option 3: Add multi-essence Flows that register collections of Flows in the TAMS API
+  * Option 3a: Reflect Source relationships inferred by Flow relationships described in Option 3 in the API
+
+## Decision Outcome
+
+Chosen option: Option 3: Add multi-essence Flows that register collections of Flows in the TAMS API, because
+
+1. This facility is needed for even the most basic use cases
+2. The TAMS API needs to be self-contained for simple, minimal configurations
+3. Adding collections at the Source level doesn't provide all the information required to describe the corresponding relationships at the Flow level
+
+Extending this to add Option 3a would be desirable, but could be added later  
```

Comment:
Looking at the implementation, isn't it 3a anyway? Although it might be that practical stores don't implement the inferred relationships and have it as a "known limitation" initially

Reply:
That was my intention - proposed implementation is 3a but implementers could choose to limit to option 3.  Could change the ADR to make this clearer.